### DISCUSSION TITLE
[FR] Change migrations table name

### DISCUSSION THREAD

**Discussion by frederikhors** [SeaQL/sea-orm#1197] [Ideas]
Hi guys, I searched all over the places but I cannot find any docs/post/issue on how to change migrations table name.

Now I'm using

```rust
Migrator::up(&postgres_connection, None).await.unwrap();
```

and it works: creates the table `seaql_migrations`.

How to change `seaql_migrations` table name to a custom one?

**Comment by teenjuna** [comment]
Looks like it's hardcoded: https://github.com/SeaQL/sea-orm/blob/43495de0aa35fb80830ece7c858963a7fc1828e1/sea-orm-migration/src/seaql_migrations.rs#L4

**Comment by billy1624** [comment]
Hey @frederikhors, sorry for the delay. @teenjuna is right. Now the name of the migration table  is hardcoded. Can I ask why you want to customize the name of it?

**Reply by frederikhors** [reply]
@tyt2y3 I think this should be an issue for the future.

**Comment by frederikhors** [comment]
For security reasons.

**Comment by tyt2y3** [comment]
Nice to have, but not critical.

**Reply by frederikhors** [reply]
But please re-open the issue.
